src/patternfly/components/Page/examples/Page.md

```diff
@@ -77,6 +77,93 @@ import './Page.css'
 {{/page}}
 ```
 
+### Using flex layout
```

Comment:
do we need this example? it's more of a demo, and you have a demo for it, too

Reply:
No, it's demonstrative of the alternate integration method that is the user generating their own layout